packages/react/src/components/ui/FreemiumBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import type { FC } from 'react';
+import React from 'react';
+
+import { CorbadoLogoIcon } from './icons/Icons';
+
+export const FreemiumBadge: FC = () => {
+  return (
+    <div
```

Comment:
currently the badge looks like this:
![current](https://github.com/corbado/javascript/assets/134682658/76594e8f-0622-410c-acfc-7cf4ea499292)

it should be smaller and look more like this:
![expected](https://github.com/corbado/javascript/assets/134682658/4754906f-844a-4252-8628-cfc0f32bc3fe)

=> let's change these settings
outer div
height: 2rem;
width: 11rem;
border-radius: 0px 0px 1.5rem 1.5rem;

inter div:
font-size: 0.6rem;
font-weight: 700;